src/Export_Command.php

```diff
@@ -108,6 +111,7 @@ public function __invoke( $_, $assoc_args ) {
 			'end_date'          => NULL,
 			'post_type'         => NULL,
 			'post_type__not_in' => NULL,
+			'limit'             => -1,
```

Comment:
Could we use `NULL` here instead of `-1`? `NULL` is a more consistent pattern with the rest of the file.

Reply:
Based on the failed tests, it appears the warning also needs to accommodate for the default value.

Reply:
Current code validation mechanism does not let `check_*()` functions distinguish between default value and value from user input.